src/geometry/adjoints.jl

```diff
@@ -49,3 +49,50 @@ function ChainRulesCore.rrule(::Type{SVector{N, T}}, x...) where {N, T}
     end
     return y, svector_const_pb
 end
+
+function ChainRulesCore.rrule(::typeof(ImageTransformations._getindex), A::AbstractExtrapolation, x)
+    y = A(Tuple(x)...)
+    function _getindex_pb(Δy)
+        # Δy :: NamedTuple{(:r, :g, :b)} or something similar
+        Δy = eltype(A)(Δy...)
+        gr = Interpolations.gradient(A, Tuple(x)...)
```

Comment:
Why do you need a `Tuple` here? If you want to splat `x`, just splat it directly?